cmake/PythonFixUpTclTk.py

```diff
@@ -100,46 +101,80 @@ def get_linked_libraries(p: Path):
 
 if __name__ == "__main__":
 
-    if platform.system() != "Darwin":
+    if platform.system() != "Darwin" and platform.system() != 'Linux':
         sys.exit(0)
 
     print("PYTHON: Copying and fixing up Tcl/Tk")
 
     if len(sys.argv) == 2:
         python_dir = Path(sys.argv[1])
     else:
-        print("Must call " + sys.argv[0] + "with one command line argument: the path to the python_lib directory")
+        print("Must call " + sys.argv[0] + " with one command line argument: the path to the python_lib directory")
         sys.exit(1)
 
     assert python_dir.is_dir()
     lib_dynload_dir = python_dir / "lib-dynload"
 
-    tk_so = locate_tk_so(python_dir)
-    tcl_tk_sos = [Path(t["libname"]) for t in get_linked_libraries(tk_so) if "libt" in t["libname"]]
-
-    any_missing = False
-    for tcl_tk_so in tcl_tk_sos:
-        new_tcl_tk_so = lib_dynload_dir / tcl_tk_so.name
-        if str(tcl_tk_so).startswith('@loader_path'):
-            assert new_tcl_tk_so.is_file(), f"{new_tcl_tk_so} missing when the tkinter so is already adjusted. Wipe the dir"
-            print("Already fixed up the libtcl and libtk, nothing to do here")
-            continue
-        elif not str(tcl_tk_so).startswith('@') and not tcl_tk_so.exists():
-            print(f"Hmmm... could not find the dependency shared object at {tcl_tk_so}; script will continue but fail")
-            any_missing = True
-            continue
-        shutil.copy(tcl_tk_so, new_tcl_tk_so)
-        # during testing, the brew installed tcl and tk libraries were installed without write permission
-        # the workaround was to manually chmod u+w those files in the brew install folder
-        # instead let's just fix them up once we copy them here
-        current_perms = os.stat(str(new_tcl_tk_so)).st_mode
-        os.chmod(str(new_tcl_tk_so), current_perms | stat.S_IWUSR)
-        # now that it can definitely be written, we can run install_name_tool on it
-        subprocess.check_output(
-            ["install_name_tool", "-change", str(tcl_tk_so), f"@loader_path/{new_tcl_tk_so.name}", str(tk_so)]
-        )
-        # Change the id that's the first line of the otool -L in this case and it's confusing
-        subprocess.check_output(["install_name_tool", "-id", str(new_tcl_tk_so.name), str(new_tcl_tk_so)])
-
-    if any_missing:
-        sys.exit(1)
+    if platform.system() == 'Darwin':
```

Comment:
If we are on Mac, we haven't changed anything from 24.2.  We still find the tk blobs, move things around, and run install_name_tool to get everyone happy.